lib/emque/consuming/version.rb

```diff
@@ -1,5 +1,5 @@
 module Emque
   module Consuming
-    VERSION = "1.0.0.beta4"
+    VERSION = "1.0.0.beta5"
```

Comment:
worthy of a full 1.1.0? Feels like a significant new feature. But, that's a weird jump I realize.